.travis.yml

```diff
@@ -1,19 +1,19 @@
 # In .travis.yml
 gemfile:
-  - gemfiles/3.0.20.gemfile
-  - gemfiles/3.1.12.gemfile
-  - gemfiles/3.2.19.gemfile
-  - gemfiles/4.0.10.gemfile
-  - gemfiles/4.1.6.gemfile
+  - gemfiles/rails_4.2.11.gemfile
+  - gemfiles/rails_5.0.7.1.gemfile
+  - gemfiles/rails_5.1.6.1.gemfile
+  - gemfiles/rails_5.2.1.1.gemfile
 
 language: ruby
 rvm:
-  - "1.9.3"
-  - "2.0.0"
-  - "2.1.3"
+  - 2.2
+  - 2.3
+  - 2.4
+  - 2.5
```

Comment:
Those are the Ruby versions supported by the currently maintained Rails versions.